### PR TITLE
Add IP6 support

### PR DESCRIPTION
**Pull request by lafka**
Todays code does not check or resolve ip6 addresses.

* Check if Hostname is an IP address before trying inet:getaddr/2
* Try resolving inet6 addresses then fallback to inet4

### CODE REVIEW

File: src/influx_udp_worker.erl [hunk 1]
```diff
@@ -86,6 +87,9 @@ code_change(_OldVsn, State, _Extra) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+addr_family({_,_,_,_}) -> inet;
+addr_family({_,_,_,_,_,_,_,_}) -> inet6.
```
Comment (elvisci) [inline src/influx_udp_worker.erl]: Missing space after "," on line 91

File: src/influx_udp_worker.erl [hunk 2]
```diff
@@ -86,6 +87,9 @@ code_change(_OldVsn, State, _Extra) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+addr_family({_,_,_,_}) -> inet;
```
Comment (elvisci) [inline src/influx_udp_worker.erl]: Missing space after "," on line 90

**Comment by palkan** [comment]
Looks good! Just fix style a little according to Elvis comments.
Thanks!

**Comment by lafka** [comment]
Done!

**Comment by palkan** [comment]
Great! And please squash commits so I can merge it.

**Pull request merged**